python/az/aro/azext_aro/_aad.py

```diff
@@ -51,6 +54,12 @@ def create_application(self, display_name):
 
         return app, password
 
+    def set_managed_app_url(self, cli_ctx):
```

Comment:
This might be "pythonizm" but this is ephemeral call, and we setting this as global variable.
Why not just, and avoid all the variables at all? 
```
identifier_uris=[
   self.getManagedApplicationURL(cli_ctx)
]
```

Reply:
Good suggestion, MJ. Updated PR to include your suggested changes. We were at this point earlier in the change but forgot to just keep it simple :sweat_smile: